### ISSUE TITLE
Feature request: Make default duration of an event a variable

### ISSUE THREAD

**Issue by vnckppl** [kidd/org-gcal.el#184]
Currently, the default duration is set to 5 minutes. Could this be converted into a variable that the user can set in the config?

**Comment by telotortium** [comment]
Yes, feel free to send a PR, otherwise I'll get to it at some point, but no guarantees on when.
